src/m_pd.h

```diff
@@ -61,8 +61,14 @@ extern int pd_compatibilitylevel;   /* e.g., 43 for pd 0.43 compatibility */
 #include <stddef.h>     /* just for size_t -- how lame! */
 #endif
 
-/* Microsoft Visual Studio is not C99, it does not provide stdint.h */
-#ifdef _MSC_VER
+/* Microsoft Visual Studio is not C99. However it started adding C99 headers
+   beginning with VS2015: https://en.wikipedia.org/wiki/Microsoft_Visual_C%2B%2B#C */
+#if !defined(_MSC_VER) || _MSC_VER >= 1900
```

Comment:
So this might be worth discussing. You've been defining your own int types if MSC_VER is found on the assumption MSVC doesn't have stdint.h. However, MSVC has had stdint.h since Visual Studio 2015. Instead of checking for MSC_VER, you could check for the presence of an MSC_VER less than 1900. This would make it easier to include m_pd.h in files that independently include stdint when embedding (which the C++ program I'm using to test this PR does).

Would this be an acceptable change? Should it be a separate PR? (I assume it should at least be its own commit.)

(Let's leave aside for the moment that you probably don't need bool or ssize_t, as already discussed.)

Reply:
> Would this be an acceptable change? Should it be a separate PR?

I've noticed this myself and I agree with you, but that could be a seperate PR (probably even on the develop branch). Only make changes that are necessary for task at hand and safe the rest for later. The smaller the change set, the bigger the chance that things get merged :-)